### PR TITLE
Update travis

### PR DESCRIPTION
**Pull request by goosemania**
[noissue]

### CODE REVIEW

File: .travis/post_before_script.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ mongo pulp_database --eval 'db.createUser({user:"travis",pwd:"travis",roles:["re
 
 # Otherwise, pulpcore will get installed as the stable release from PyPI
 # as a dep of pulp-2to3-migration.
-pip install $TRAVIS_BUILD_DIR/../pulpcore
+#pip install $TRAVIS_BUILD_DIR/../pulpcore
 
-pip install .
+#pip install .
```
Comment (goosemania) [inline .travis/post_before_script.sh:21]: @dkliban , I'm trying to figure out why it needs to be installed outside of  the container, any hints?

File: pulp_2to3_migration/tests/functional/test_file_migration.py [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
 
 # Initialize MongoDB connection
-initialize()
+# initialize()
```
Comment (goosemania) [inline pulp_2to3_migration/tests/functional/test_file_migration.py:32]: @dkliban @dralley , why do we need to initialize a mongodb connection from the functional tests code? How do we use it? I thought that Pulp which runs in a container does it when we trigger a task to migrate from pulp 2 to pulp 3.

Comment (goosemania) [reply pulp_2to3_migration/tests/functional/test_file_migration.py:32]: Found it. We use mongo to find the list of repositories in pulp2.

File: .travis/post_before_script.sh (hunk 1 above)
Comment (goosemania) [reply .travis/post_before_script.sh:21]: It seems that we need configs for functional tests  from both pulpcore settings and pulp-2to3-migration settings

**Review by ipanova — APPROVED** [review]

**Pull request merged**